src/plugins/controls/public/options_list/components/options_list_editor_options.tsx

```diff
@@ -39,16 +42,41 @@ export const OptionsListEditorOptions = ({
           }}
         />
       </EuiFormRow>
-      <EuiFormRow>
+      <EuiFormRow display="columnCompressedSwitch">
         <EuiSwitch
-          label={OptionsListStrings.editor.getRunPastTimeoutTitle()}
-          checked={Boolean(state.runPastTimeout)}
+          label={OptionsListStrings.editor.getAllowExclude()}
+          checked={!state.allowExclude}
           onChange={() => {
-            onChange({ runPastTimeout: !state.runPastTimeout });
-            setState((s) => ({ ...s, runPastTimeout: !s.runPastTimeout }));
+            onChange({ allowExclude: !state.allowExclude });
+            setState((s) => ({ ...s, allowExclude: !s.allowExclude }));
           }}
         />
       </EuiFormRow>
+      <EuiFormRow>
+        <EuiFlexGroup alignItems="center" gutterSize="xs">
+          <EuiFlexItem grow={false}>
+            <EuiSwitch
+              label={OptionsListStrings.editor.getRunPastTimeoutTitle()}
+              checked={Boolean(state.runPastTimeout)}
+              onChange={() => {
+                onChange({ runPastTimeout: !state.runPastTimeout });
+                setState((s) => ({ ...s, runPastTimeout: !s.runPastTimeout }));
+              }}
+            />
+          </EuiFlexItem>
+          <EuiFlexItem
+            grow={false}
+            css={css`
+              margin-top: 0px !important;
+            `}
+          >
+            <EuiIconTip
+              content={OptionsListStrings.editor.getRunPastTimeoutTooltip()}
+              position="right"
+            />
+          </EuiFlexItem>
```

Comment:
Since I was in the editor options component anyway to add the new toggle, I figured I would knock out https://github.com/elastic/kibana/issues/134369 as well since it's a straightforward change. This is what the new text + tooltip currently looks like for the `"Ignore timeout"` setting:

![Oct-24-2022 10-10-20](https://user-images.githubusercontent.com/8698078/197574352-df664261-4f2e-49f1-bf4a-7bb75d1b31d1.gif)

Reply:
@Heenawter I think this is fine :)

Reply:
@andreadelrio Amazing, thank you so much 🙇